docs/_data/table/common-configuration.csv

```diff
@@ -99,6 +99,7 @@ alluxio.underfs.s3a.streaming.upload.enabled,"false"
 alluxio.underfs.s3a.streaming.upload.partition.size,"64MB"
 alluxio.web.file.info.enabled,"true"
 alluxio.web.resources,"${alluxio.home}/core/server/common/src/main/webapp"
+alluxio.web.resources,"${alluxio.home}/alluxio-ui/"
```

Comment:
Do we need to delete the previous line? Also this file is usually autogenerated.

Reply:
Yes, we do, thank you. Although now that I know it's autogenerated, I will revert this change.